scripts/coverage

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+export PREFIX=""
+if [ -d 'venv' ] ; then
+    export PREFIX="venv/bin/"
+fi
+export SOURCE_FILES="httpx tests"
+
+set -x
+
+${PREFIX}coverage report --show-missing --skip-covered --fail-under=97
```

Comment:
I can't find `show-missing` on the coverage.py help or docs?

Reply:
In the docs it’s only referred to by `-m`

Reply:
Ah, I see 👍